justfile

```diff
@@ -0,0 +1,21 @@
+# justfile for common project commands
+# see here for more information: https://github.com/casey/just
+
+# find system default shell
+hashbang := if os() == 'macos' {
+	'#!/usr/bin/env zsh'
+} else {
+	'#!/usr/bin/env bash'
+}
+
+# show a list of just commands for this project
+default:
+  @just --list
+
+# setup conda envs
+@setup-conda-envs:
+    #!{{hashbang}}
+    conda remove -n jump_sc --all
+    conda remove -n R_jump_sc --all
```

Comment:
Why have these remove sections? Why not just only have the install environments?

Reply:
Thanks for this - it's a good question! I've encountered behavior from conda in the past that made it so any changes to the `environment.yml` could potentially corrupt or have erroneous behavior when modifying the existing environments. The removals here were a matter of trying to be safe about this. That said, I might be being overprotective here, and will shift to use something like `conda env update --name my_env --file environment.yml --prune` instead.